cmd/svcat/output/class.go

```diff
@@ -49,6 +50,56 @@ func WriteClassList(w io.Writer, classes ...v1beta1.ClusterServiceClass) {
 	t.Render()
 }
 
+func writeClassListJSON(w io.Writer, classes []v1beta1.ClusterServiceClass) {
+	classList := v1beta1.ClusterServiceClassList{
+		Items: classes,
+	}
+	j, _ := json.MarshalIndent(classList, "", "   ")
+	w.Write(j)
+}
+
+func writeClassListYAML(w io.Writer, classes []v1beta1.ClusterServiceClass) {
+	classList := v1beta1.ClusterServiceClassList{
+		Items: classes,
+	}
+	y, _ := yaml.Marshal(classList)
+	w.Write(y)
+}
+
+// WriteClassList prints a list of classes in the specified output format.
+func WriteClassList(w io.Writer, outputFormat string, classes ...v1beta1.ClusterServiceClass) {
+	switch outputFormat {
+	case "json":
+		writeClassListJSON(w, classes)
+	case "yaml":
+		writeClassListYAML(w, classes)
+	case "table":
+		writeClassListTable(w, classes)
+	}
+}
+
+func writeClassJSON(w io.Writer, class v1beta1.ClusterServiceClass) {
+	j, _ := json.MarshalIndent(class, "", "   ")
```

Comment:
Ok, yeah we should definitely switch over to using the same pattern already set by `writeYaml` (and add a `writeJson` too). It will reduce all the yaml imports, handles indenting across the board and has error handling.

Reply:
Yeah, I agree. I originally *didn't* do this because I was making those crufty things for the types.